### DISCUSSION TITLE
Make WindiCSS working with Vite when setting root option in vite.config.js

### DISCUSSION THREAD

**Discussion by loctran016** [windicss/windicss#442] [Q&A] [answered]
Hi. I'm having trouble migrating from TailwindCSS.

* Here is my [source code (migrating to WindiCSS)](https://github.com/loctran016/result/tree/windicss)
* Here is what it look like in [TailwindCSS](https://loc-result.netlify.app/)
* Here is what  it look like in [WindiCSS](https://deploy-preview-1--loc-result.netlify.app/?utm_source=github&utm_campaign=bot_dp)

Even though I've set like this in [windi.config.ts](https://github.com/loctran016/result/blob/windicss/windi.config.ts#L15-L17):
```ts
import { defineConfig } from 'windicss/helpers';
export default defineConfig({
  //   ...
  theme: {
    extend: {
      gridTemplateColumns: {
        layout: 'repeat(auto-fill, minmax(15rem,1fr))',
      },
    },
    //   ...
  },
});
```
But when I add class `.grid-cols-layout` to the [card section](https://github.com/loctran016/result/blob/windicss/src/index.html#L142), it doesn't work. It used to work in tailwind.config.js
Any help is welcome. Thanks in advanced.

Edit: It's just like WindiCSS doesn't read the configuration file, change other setting and doesn't work as well.

**Comment by antfu** [comment]
Can you share your repo?

**Reply by loctran016** [reply]
Oh, sorry. I've put a wrong link. My repo is [here](https://github.com/loctran016/result/tree/windicss). It's in the branch `windicss`

**Comment by loctran016** [comment, marked as answer]
I finally figured this out. I must put it in the root folder (in my case is inside `src` because of [this line](https://github.com/loctran016/result/blob/windicss/vite.config.js#L6)). May I create a PR updating the doc about this problem? @antfu
